src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -45,10 +45,11 @@
      * @throws FormatException when a file has a syntax error
      * @throws PathException   when a file does not exist or is not readable
      */
-    public function load($path/*, ...$paths*/)
+    public function load($path, ...$paths)
```

Comment:
imo it's not that bad if the ...$paths is empty. No file loaded, no env variable created